kafka_utils/kafka_consumer_manager/util.py

```diff
@@ -181,19 +183,19 @@ def read_group(self, group_id):
 
     def read_groups(self, partition=None):
         self.log.info("Kafka consumer running")
-        if partition is not None:
-            topic_partition = {CONSUMER_OFFSET_TOPIC: [partition]}
-        else:
-            topic_partition = CONSUMER_OFFSET_TOPIC
-
         self.consumer = KafkaConsumer(
-            topic_partition,
             group_id='offset_monitoring_consumer',
             bootstrap_servers=self.kafka_config.broker_list,
-            auto_offset_reset='smallest',
-            auto_commit_enable=False,
+            auto_offset_reset='earliest',
+            enable_auto_commit=False,
             consumer_timeout_ms=3000,
         )
+
+        if partition:
```

Comment:
Truthy check of partition value will not work here. When partition value is zero, it evaluates to False.